src/components/Form.js

```diff
@@ -147,7 +149,21 @@ const reducer = (draft, action) => {
       }
     });
     history.push('/bevestiging');
-  }
+  };
+
+  const onLogout = async (event) => {
+    event.preventDefault();
+
+    const confirmationQuestion = intl.formatMessage(
+      {id: 'confirmLogout', defaultMessage: 'Are you sure that you want to logout?'}
+    );
+    if (!window.confirm(confirmationQuestion)) {
+      return;
+    }
+    await destroy(`${config.baseUrl}authentication/session`);
+    history.push('/');
+    window.location.reload();
```

Comment:
fine for now - but I think later on it'd be cleaner to dispatch an action to reset the state completely without having to do a full page reload.